src/chart/line/LineView.ts

```diff
@@ -730,6 +730,10 @@ class LineView extends ChartView {
                 polygon = this._newPolygon(
                     points, stackedOnPoints
                 );
+            }// If areaStyle is removed
+            else if(polygon){
```

Comment:
Hi, @jiawulin001 Thanks for your recent continuous contribution! 👍

I just scanned your three PRs(#16719, #16750, and this) and found some code style flaws that may bring lint errors. There is a [coding standard](https://echarts.apache.org/zh/coding-standard.html) for reference on the website. It should be useful for you.

Reply:
Hi, @plainheart thank you for scanning code for me. I've updated the code of two open PRs and do I need to submit another PR to update the code in the first PR? Just out of curiosity, what lint tool are you using? I am using sonarlint and it doesn't seem to bother reminding me of any of this.

Reply:
> do I need to submit another PR to update the code in the first PR? 

No need to do that. I fixed it in the release of v5.3.2.

> what lint tool are you using?

No tools. Just scan with my eyes. Here **"_scan_"** means I looked through it quickly and didn't check the logic.

Reply:
> It seems to be not working for you.
I am not sure... Maybe it's better to add it to github workflow? Anyway, I will be careful about the code style, thanks for your suggestion.